docs/en/week07/07-2.md

```diff
@@ -0,0 +1,173 @@
+<!-- ---
+lang-ref: ch.07-2
+title: 
+authors: Ravi Choudhary， B V Nithish Addepalli, Syed Rahman，Jiayi Du
+date: 16 Mar 2020
+--- -->
+
+
+[comment]: # (lang-ref: ch.07-2)
+[comment]: # (Time: 00:51:00 - 1:06:00)
+[comment]: # (Author: Ravi Choudhary)
```

Comment:
WHAT IS THIS???
Have you run the website locally?
You just commented out what is needed to actually get the website to render!!!

Com'on, people!